tests/Microsoft.Identity.Test.Common/Core/Helpers/OTelInstrumentationUtil.cs

```diff
@@ -0,0 +1,136 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using Microsoft.Identity.Client.Platforms.Features.OpenTelemetry;
+using Microsoft.Identity.Client.TelemetryCore;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+
+namespace Microsoft.Identity.Test.Common.Core.Helpers
+{
+    public class OTelInstrumentationUtil
+    {
+        public static void VerifyActivity(int expectedTagCount, List<Activity> _exportedActivities)
+        {
+            Assert.AreEqual(1, _exportedActivities.Count);
```

Comment:
Shouldn't there be 5 activities in the collection - 2 successful and 3 failed ones because there were 5 requests?

Reply:
Right, the activity should start at the beginning of each request. Updated.